pom.xml

```diff
@@ -38,7 +38,7 @@
     </scm>
 
     <properties>
-        <es.version>7.9.3</es.version>
+        <es.version>7.0.1</es.version>
```

Comment:
We reverted the version upgrade from here: https://github.com/confluentinc/kafka-connect-elasticsearch/pull/557/files until we can confirm that it will not break compatibility